package-builder/flox-build.mk

```diff
@@ -421,7 +422,8 @@ define BUILD_local_template =
 	@# Actually perform the build using the temporary build wrapper.
 	@#
 	@echo "Building $(_name) in local mode"
-	$(_VV_) $(_rm) -rf $($(_pvarname)_out)
+	-$(_VV_) $(_find) $($(_pvarname)_out) -type d -exec $(_chmod) +w {} \;
+	-$(_VV_) $(_rm) -rf $($(_pvarname)_out)
```

Comment:
(non-blocking) Is it intentional that some of these are chained as `find && rm` but some aren't as `find; rm`?